Cognite.Logging/Logger.cs

```diff
@@ -200,15 +212,20 @@ public void Enable()
     /// Extension utilities for logging
     /// </summary>
     public static class LoggingExtensions {
-        
+
         /// <summary>
         /// Adds a configured Serilog logger as singletons of the <see cref="Microsoft.Extensions.Logging.ILogger"/> and
         /// <see cref="Serilog.ILogger"/> types to the <paramref name="services"/> collection.
         /// A configuration object of type <see cref="LoggerConfig"/> is required, and should have been added to the
         /// collection as well.
         /// </summary>
         /// <param name="services">The service collection</param>
-        public static void AddLogger(this IServiceCollection services) {
+        /// <param name="buildLogger">Method to build the logger.
+        /// This defaults to <see cref="LoggingUtils.GetConfiguredLogger(LoggerConfig)"/>,
+        /// which creates logging configuration for file and console using
+        /// <see cref="LoggingUtils.GetConfiguration(LoggerConfig)"/></param>
+        public static void AddLogger(this IServiceCollection services, Func<LoggerConfig, Serilog.ILogger>? buildLogger = null) {
+            if (buildLogger == null) buildLogger = LoggingUtils.GetConfiguredLogger;
```

Comment:
```suggestion
            buildLogger ??= LoggingUtils.GetConfiguredLogger;
```

;)

not sure if it's more readable, though